coursefinder/templates/coursefinder/course_finder_choose_country.html

```diff
@@ -0,0 +1,22 @@
+{% extends "coursefinder/course_finder_base.html" %}
+{% load static %}
+{% load wagtailcore_tags %}
+
+
+{% block question %}
+<h3>{{page.question}}</h3>
+<form>
+  <div  class="checkbox">
```

Comment:
I would tend to keep lists in alphabetical order on each of our templates if there is not a defined ordering; which i believe is the case here.